app/views/includes/fileProcessingAlert.scala.html

```diff
@@ -0,0 +1,16 @@
+
+@this(
+        govukWarningText: GovukWarningText
+)
+
+@()(implicit messages: Messages)
+
+<div class="govuk-!-display-none" id="file-processing-alert">
+    <div class="govuk-!-margin-bottom-2" style="display: flex; justify-content: center; align-items: center">
+        <img id="file-processing-alert-img" src="@routes.Assets.versioned("images/spinner.gif")" alt="Loading symbol"/>
```

Comment:
hate to make you wait longer but I think this `Loading symbol` text will also need to be in messages